src/neuralnet/superblock.h

```diff
@@ -0,0 +1,248 @@
+#pragma once
+
+#include "scraper/fwd.h"
+#include <string>
+#include "key.h"
+#include "neuralnet/cpid.h"
+
```

Comment:
We are missing `serialize.h` here. It happens to work for now as it gets it somewhere in the include chain.

Reply:
✔️